.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Publish
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+
+      - name: Set settings.xml
+        uses: s4u/maven-settings-action@v2.8.0
+        with:
+          servers: |
+            [{
+              "id": "mks-nexus-private",
+              "username": "${{ secrets.MEKOM_NEXUS_USERNAME }}",
+              "password": "${{ secrets.MEKOM_NEXUS_PASSWORD }}"
+            },
+            {
+              "id": "mks-nexus-private-snapshots",
+              "username": "${{ secrets.MEKOM_NEXUS_USERNAME }}",
+              "password": "${{ secrets.MEKOM_NEXUS_PASSWORD }}"
+            }]
+        if: ${{ github.ref == 'refs/heads/main' }}
+
+      - name: Build and publish (with 'prod' profile)
```

Comment:
There isn't any prod profile for Bahmni Docker Compose. Command should be `mvn --batch-mode clean deploy`